file-formats/dras/type/zif_aff_dras_v1.intf.abap

```diff
@@ -0,0 +1,16 @@
+INTERFACE zif_aff_dras_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">CDS Aspect</p>
+    "! CDS scalar aspect
+    BEGIN OF ty_main,
+      "! $required
+      format_version TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header         TYPE zif_aff_types_v1=>ty_header_60,
```

Comment:
Just a short question? Is key-user-tooling supported for DRAS?

Reply:
Not yet.

Reply:
OK, If you do not use key-user tooling I suggest to use `ty_header_60_cloud` which only supports "standard" and "cloudDevelopmnet"

```suggestion
      header         TYPE zif_aff_types_v1=>ty_header_60_cloud,
```

Reply:
It will be supported in the future, though. Is it preferable to have it already prepared or should it be restricted now and changed back to `ty_header_60` later?

Reply:
I suggest to change it and change it back to `ty_header_60` as soon as it is supported. As far as I can see this change should be compatible (in the future)